views/partials/postfooter.handlebars

```diff
@@ -0,0 +1,36 @@
+<!-- Footer -->
```

Comment:
I like the use of partials here! Generally speaking headers and footers should be the same across the entire site! What's your reason for having a separate header for your post page?